rules/linux/persistence_init_d_file_creation.toml

```
@@ -68,7 +68,7 @@ This rule looks for the creation of new files within the `/etc/init.d/` director
 
 > **Note**:
 > This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will display unrendered Markdown in this guide.
-
+> This investigation guide leverages functionality that allows the usage of variables within the OSQuery queries. These variables will be directly populated by the corresponding values within the detection alert at run time. This feature has been introduced in Elastic Stack version 8.7.0. Older Elastic Stack versions will not be able to leverage this feature, and will require manual adjustments to run the query.
```

Just some slight modifications to keep the language consistent with our [feature docs](https://www.elastic.co/guide/en/security/current/osquery-placeholder-fields.html) and the format of the previous note:

```suggestion
> This investigation guide uses [placeholder fields](https://www.elastic.co/guide/en/security/current/osquery-placeholder-fields.html) to dynamically pass alert data into Osquery queries. Placeholder fields were introduced in Elastic Stack version 8.7.0. If you're using Elastic Stack version 8.6.0 or earlier, you'll need to manually adjust this investigation guide's queries to ensure they properly run.   
```